mods/d2k/rules/structures.yaml

```diff
@@ -1173,7 +1173,7 @@ palace:
 		OrderName: ProduceActorPower.Fremen
 	ProduceActorPower@saboteur:
 		Description: Recruit Saboteur
-		LongDesc: Sneaky infantry, armed with explosives\n  Strong vs Buildings\n  Weak vs Everything\n  Special Ability: destroy buildings
+		LongDesc: Sneaky infantry, armed with explosives.\nCan be deployed to become invisible for a while.\n  Strong vs Buildings\n  Weak vs Everything\n  Special Ability: destroy buildings
```

Comment:
Please change this to something like "Can be deployed to become invisible for a limited time."